src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -141,6 +189,22 @@ def retrieve_download_links(
             cloud_coverage_whole,
         )
 
+        if find_nearest_to_previous_link is True:
```

Comment:
Suggestion: The `is True` is redundant:

Just simplify this to `if find_nearest_to_previous_link`